velox/functions/prestosql/tests/IPAddressFunctionsTest.cpp

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/common/base/tests/GTestUtils.h"
+#include "velox/functions/prestosql/tests/utils/FunctionBaseTest.h"
+
+namespace facebook::velox::functions::prestosql {
+class IPAddressFunctionsTest : public functions::test::FunctionBaseTest {
+ protected:
+  std::optional<std::string> ipPrefixFunctionFromIpAddress(
+      const std::optional<std::string>& input,
+      const std::optional<int64_t>& mask) {
+    return evaluateOnce<std::string>(
+        "cast(ip_prefix(cast(c0 as ipaddress), c1) as varchar)", input, mask);
+  }
+
+  std::optional<std::string> ipPrefixFromVarChar(
+      const std::optional<std::string>& input,
+      const std::optional<int64_t>& mask) {
+    return evaluateOnce<std::string>(
+        "cast(ip_prefix(c0, c1) as varchar)", input, mask);
+  }
```

Comment:
Are `ipPrefixFunctionFromIpAddress` and `ipPrefixFromVarChar` mixed up and should be the other way around?